packages/permissionless/accounts/biconomy/signerToBiconomySmartAccount.ts

```diff
@@ -122,14 +121,11 @@ const getAccountInitCode = async ({
     })
 
     // Build the account init code
-    return concatHex([
-        factoryAddress,
-        encodeFunctionData({
-            abi: createAccountAbi,
-            functionName: "deployCounterFactualAccount",
-            args: [ecdsaModuleAddress, ecdsaOwnershipInitData, index]
-        }) as Hex
-    ])
+    return encodeFunctionData({
```

Comment:
this is an internal function, in the new v0.7 we need factor & factory data separately so now this only returns factory data instead of initCode( factory + factory Data). so this function can be used both for v0.7 and v0.6, in v0.6 we append factory where we call this internal function.